### DISCUSSION TITLE
Accurite 609TXC changes ID dynamically?

### DISCUSSION THREAD

**Discussion by eigenphase** [merbanan/rtl_433#2481] [Q&A] [answered]
I thought IDs for Acurite temperature sensors are fixed per device but it seems they "can" change. Did anyone experience this yet? Under which conditions does it change?

I have one sensor which had ID=20 for two months (since I got it). Today all of a sudden I saw no results were received for 4 hours. 
In MQTTExplorer I saw that ID seem to have changed to 163!

**Comment by rct** [comment, marked as answer]
Acurite sells both sensors with permanent IDs and volatile IDs. The 609 is one of the volatile ones.   The devices that have channel A/B/C switches and are compatible with "My Acurite" have permanent IDs

If the battery drops low enough, the device may reset and generate a new ID.
